decode.go

```diff
@@ -49,6 +64,16 @@ func NewDecoder(b []byte) *Decoder {
 	return &Decoder{reader: bufio.NewReader(bytes.NewReader(b)), typeRefs: &TypeRefs{records: map[string]bool{}}}
 }
 
+// NewStrictDecoder generate a strict mode decoder instance.
+// In strict mode, all target class must be registered.
```

Comment:
generate -> generates

pls add this comment to readme to let the user know how to use it.